lib/vdsm/tool/config_lvm_filter.py

```diff
@@ -122,6 +123,47 @@ def config_with_filter(
         return CANNOT_CONFIG
 
 
+def config_with_devices(args, mounts, current_wwids, wanted_wwids):
+
+    # Check if the lvm devices is already configured.
+    if lvmdevices.is_configured():
+        print("LVM devices already configured for vdsm")
+        return
+
+    # Before creating devices file we have to also configure multipath
+    # blacklist.
+
+    if current_wwids != wanted_wwids:
+        _print_wanted_blacklist(wanted_wwids)
+
+        if not args.assume_yes:
+            if not common.confirm("Configure host? [yes,NO] "):
+                return NEEDS_CONFIG
+
+        mpathconf.configure_blacklist(wanted_wwids)
+
+    # Find VGs of mounted devices.
+    vgs = set()
+    for mnt in mounts:
+        vgs.add(mnt.vg_name)
+
+    # Multipath blacklist is configured at this point. Configure lvm to use
+    # devices file and create initial devices file. As vgimportdevices takes
+    # into account existing lvm filter and we don't wan't to depend on it,
+    # specify our own filter which allows all the devices. This avoids issues
+    # with misconfigured custom filters. As we take into account only mounted
+    # devices and lvm filter provided by vdsm is constructed from the same
+    # information, there shouldn't be any glitch with correct vdsm filter.
+
+    lvmdevices.enable()
+    lvmdevices.import_vg_devices(
+        vgs, config="devices { filter = [\"a|.*|\"] }")
```

Comment:
Why do we have to specify the filter here when there is only one filter that makes sense?

Reply:
As the filter can be wrong and we may fail to find the right filter. This is how I implemented it in the previous version and you suggested to not take care about the filter and enable everything, see https://github.com/oVirt/vdsm/pull/27#discussion_r782435968

So how it should be implemented? Let vdsm construct the filter and use it without asking user?

Reply:
The filer `["a|.*|"]` is fine of course, as discussed before. But since this is really the only option, why make it an argument and complicate the calling code?